tests/clientcmd/command.go

```diff
@@ -56,182 +48,30 @@ func RunCommand(cmdName string, args ...string) (string, string, error) {
 }
 
 func RunCommandWithNS(namespace string, cmdName string, args ...string) (string, string, error) {
```

Comment:
This same function is now implemented here and in https://github.com/kubevirt/kubevirt/pull/8144/commits/6332f1fc26aabe5861191cec49f0f586e8d554b8#diff-963dce28b2001e333d14801a32eec231c16eb806181fa6f15a3a8c6a6b4f7f91R42
I think it's preferred to remove this one and update the tests that call it to use the `libcli`

Reply:
The implementation is different here we do 
```golang
return libcli.RunCommandWithNSAndInput(namespace, nil, getCmdPathFromName(cmdName), args...)
```
The getCmdPathFromName depends on flags packages, the idea is to remove that dependency from reporter since it has some init functions that hurts using the reporter at other projects.

Reply:
Ahh, gotcha! My bad.